tools/test/examples/examples.json

```diff
@@ -454,6 +454,21 @@
         "baud_rate": 9600,
         "compare_log": ["mbed-os-example-atecc608a/tests/atecc608a.log"],
         "auto-update" : true
+    },
+    {
+      "name": "mbed-os-example-aws",
+      "github": "https://github.com/ARMmbed/mbed-os-example-aws",
+      "sub-repo-example": false,
+      "subs": [],
+      "features" : [],
+      "targets" : [],
```

Comment:
Should we limit the targets? For example, the previous CI run also tried to build for NUCLEO_F411RE and failed, and I don't think it's a WiFi or Ethernet target.

For the WiFi example (line 120) we only build it for one target.

Reply:
Maybe K64F (Ethernet), and one WiFi target TBC

Reply:
We should limit to a few targets with supported HW. @jamesbeyond

Reply:
Do we have a blocker created for this ? To my understanding this should be in 6.1 correct?

Reply:
This needs to go in 6.1.

Reply:
Okay if we set targets in the future, since the example is disabled for now...